### PR TITLE
Allow graphql to pass through null values instead of ignoring them

### PR DESCRIPTION
**Pull request by lprhodes**
As set out in #133 and https://github.com/facebook/graphql/pull/83

### CODE REVIEW

**Comment by urbanappetite** [comment]
I'm not seeing any trailing whitespace - any ideas?

**Comment by leebyron** [comment]
Thanks for investigating this. It looks like the linter is complaining about line 164 in values.js. I'd suggest configuring your IDE to trim trailing whitespace on save.

Right now it's very much intentional how null values are treated because of how the GraphQL spec is written. I'm still working on that spec change which will determine the runtime behavior changes.

File: src/execution/values.js [hunk 1]
```diff
@@ -148,10 +148,9 @@ function coerceValue(type: GraphQLInputType, value: any): any {
     return Object.keys(fields).reduce((obj, fieldName) => {
       var field = fields[fieldName];
       var fieldValue = coerceValue(field.type, value[fieldName]);
-      if (isNullish(fieldValue)) {
+      if (fieldValue === undefined) {
         fieldValue = field.defaultValue;
-      }
-      if (!isNullish(fieldValue)) {
+      } else {
```
Comment (leebyron) [inline src/execution/values.js]: this seems like a mistake - did your unit tests pass or are we missing a test for this condition? As I read it, if a fieldValue is coerced to be undefined, then we set `fieldValue` to the default value, however never actually set it on the response object.

**Comment by wincent** [comment]
Closing this one due to staleness.

**Pull request closed without merging**